### ISSUE TITLE
Test run in action is not complete

### ISSUE THREAD

**Issue by JakobVogelsang** [openscd/open-scd#483]
At the moment, we run a test with checkout branch when we push to a PR. I would argue that we need to extend the trigger to pushes to main otherwise a scenario like that is possible - and happened already - thanks @dlabordus for the hot-fix:

We work on two PRs. Both can be merged and test run on checkout main. We push one of them to main and do not re-run the test for the other one! The tests then give a false positive and one might and I had pushed to main with failing tests.

**Comment by dlabordus** [comment]
As far as I understood, when running on "pull_request" the main branch is merged with the feature branch, but that is done once when the pull request is created and not updated if the main branch changes when the pull request is open.

To be sure, was the pull request for the SampledValueControl created before the change to the wizards was merged to main?

**Comment by JakobVogelsang** [comment]
Yes it was. I pushed it yesterday and pushed the other on today to main.
